### DISCUSSION TITLE
Geräte periodisch einschalten möglich?

### DISCUSSION THREAD

**Discussion by teuti** [camueller/SmartApplianceEnabler#549] [Q&A]
Moin zusammen,

ich habe im Garten einen Bereich mit Rasen neu eingesät. Jetzt möchte ich gerne, dass der SAE meine Pumpe alle x Minuten für y Minuten einschaltet. Gibt es einen Schalter, den ich bisher übersehen habe, mit dem dieses Verhalten eingestellt werden kann oder muss ich mir ganze viele Zeitpläne einrichten?

Gruß,
teuti

**Comment by camueller** [comment]
So etwas gibt es bisher nicht, weil der SAE keine Zeitschaltuhr ist, sondern dem SHM ermöglichen sollen, die PV-Energie optimal zu nutzen. Bei Deinen festen Zeiten ist das nicht möglich.
